packages/react/src/label/index.ts

```diff
@@ -1,3 +1,7 @@
-export { Description, type DescriptionProps } from './Description';
+export {
+  Description,
+  type DescriptionProps,
+  descriptionClasses,
```

Comment:
Blir denne tilgjengelig for konsumenter av grunnmuren nå?

Altså `import { descriptionClasses } from '@obosbbl/grunnmuren-react'`?

Det føles ikke ut som den burde være tilgjengelig utenifra, siden dette er en midlertidig intern workaround.

Reply:
Ja, shit det har du nok rett i! Good catch

Reply:
Nei forresten, den blir ikke det. `label` blir ikke eksportert: [https://github.com/code-obos/grunnmuren/blob/main/packages/react/src/index.ts](https://github.com/code-obos/grunnmuren/blob/main/packages/react/src/index.ts)

Reply:
Men sikkert lurt å fjerne eksporten fra index.ts i tilfelle vi bestemmer oss for å eksponere dette i fremtiden.